extensions/tika/deployment/src/main/java/io/quarkus/tika/deployment/TikaProcessor.java

```diff
@@ -102,22 +113,41 @@ public void registerTikaProviders(BuildProducer<ServiceProviderBuildItem> servic
                 "META-INF/services/" + serviceProviderName));
     }
 
-    static List<String> getSupportedParserNames(Optional<String> requiredParsers) throws Exception {
+    static Map<String, List<TikaParserParameter>> getSupportedParserConfig(Optional<String> tikaConfigPath,
+            Optional<String> requiredParsers) throws Exception {
         Predicate<String> pred = p -> !NOT_NATIVE_READY_PARSERS.contains(p);
         List<String> providerNames = getProviderNames(Parser.class.getName());
-        if (!requiredParsers.isPresent()) {
-            return providerNames.stream().filter(pred).collect(Collectors.toList());
+        if (tikaConfigPath.isPresent() || !requiredParsers.isPresent()) {
+            return providerNames.stream().filter(pred).collect(Collectors.toMap(Function.identity(),
+                    p -> Collections.<TikaParserParameter> emptyList()));
         } else {
             List<String> abbreviations = Arrays.stream(requiredParsers.get().split(",")).map(s -> s.trim())
                     .collect(Collectors.toList());
-            Set<String> requiredParsersFullNames = abbreviations.stream()
-                    .map(p -> getParserNameFromConfig(p)).collect(Collectors.toSet());
+            Map<String, String> fullNamesAndAbbreviations = abbreviations.stream()
+                    .collect(Collectors.toMap(p -> getParserNameFromConfig(p), Function.identity()));
 
-            return providerNames.stream().filter(pred).filter(p -> requiredParsersFullNames.contains(p))
-                    .collect(Collectors.toList());
+            return providerNames.stream().filter(pred).filter(p -> fullNamesAndAbbreviations.containsKey(p))
+                    .collect(Collectors.toMap(Function.identity(),
+                            p -> getParserConfig(p, fullNamesAndAbbreviations.get(p))));
         }
     }
 
+    static List<TikaParserParameter> getParserConfig(String parserName, String abbreviation) {
+        final String prefix = "quarkus.tika.parsers." + abbreviation + ".";
```

Comment:
Just to be sure, when you use these properties, you don't have a bunch of warning saying the properties are unconsumed?

Reply:
@gsmet thanks for asking, actually it does:
```
[WARNING] [io.quarkus.config] Unrecognized configuration key "quarkus.tika.parsers.pdf.sort-by-position" provided
```
Hmm...The problem is the list of parsers can be nearly open ended, and if I hide it back to `tika-config.xml` then there is no warning :-), but I'd like to 'break away' from `tika-config.xml`. 

I propose to just ignore such Warnings but also consider a quarkus config enhancement to tell it not to warn for a property whose name starts from, in this case, `quarkus.tika.parsers.pdf`. @dmlloyd FYI as well.
Does it sound reasonable ?

Reply:
So the idea when we have this type of pass-through properties is to not put them in the `quarkus.` namespace.

So you would have the `quarkus.tika.` properties for everything handled by Quarkus and the `tika.` prefix for everything that is pass-through.

We already do that for a few other extensions.

Reply:
Maybe it would be better to use e.g. a `Map<String,Map<String,String>>` property than a pass-through.  You all know how I feel about pass-through properties. :)

Reply:
@gsmet, OK, thanks, the parser property configuration is now done in the custom namespace.